Chatto/Source/ChatController/ChatViewController.swift

```diff
@@ -123,11 +126,15 @@ public class ChatViewController: UIViewController, UICollectionViewDataSource, U
 
         let inputView = self.createChatInputView()
         self.inputContainer.addSubview(inputView)
-        self.inputContainer.addConstraint(NSLayoutConstraint(item: self.inputContainer, attribute: .Top, relatedBy: .Equal, toItem: inputView, attribute: .Top, multiplier: 1, constant: 0))
+        heightConstraint = NSLayoutConstraint(item: self.inputContainer, attribute: .Height, relatedBy: .Equal, toItem: nil, attribute: .NotAnAttribute, multiplier: 1, constant: 0)
+        topConstraint = NSLayoutConstraint(item: self.inputContainer, attribute: .Top, relatedBy: .Equal, toItem: inputView, attribute: .Top, multiplier: 1, constant: 0)
```

Comment:
I tried activating/deactivating the top constraint but that just changed how the top container was aligned.